### DISCUSSION TITLE
CUSTOM_BATTERY_NAMES for JKBMS BLE don't appear to work

### DISCUSSION THREAD

**Discussion by seamaster101** [Louisvdw/dbus-serialbattery#987] [General]
I have 4 JKBMS. i tried renaming them in the UI and also in the configuration file.  Either method the batteries loose their names after reboot. 
here is my syntax in  /data/etc/dbus-serialbattery/config.ini:
`BLUETOOTH_BMS = Jkbms_Ble C8:47:8C:EE:2A:E4, Jkbms_Ble C8:47:8C:EE:1E:D7, Jkbms_Ble C8:47:8C:EE:11:F7, Jkbms_Ble CUSTOM_BATTERY_NAMES = c8478cee2ae4:HB1_B1, c8478cee1ed7:HB1_B2, c8478cee11f7:HB1_B3, c8478cee18cf:HB1_B4 
`
I'm doing something worn, or is this a known issue?

**Comment by mr-manuel** [comment]
It depends on the version you are using. Check the changelog and you will know.

**Comment by seamaster101** [comment]
@mr-manuel Aha! installed the latest beta from your repository and the saving of the custom name works. 
But in an mean time i found more and quite serious problem.  I have the 4 X JKBMS connected via bluetooth, but quite often they do freeze and present stagnant data. this is definitely a big problem as the driver does not detect that there is something wrong.
<img width="617" alt="image" src="https://github.com/Louisvdw/dbus-serialbattery/assets/7897023/cab53cae-b9fc-4ecb-88da-dab93e600a2f">
Note the difference in the current - only one sensor displays proper current for the moment. the other 3 batteries the current is stuck at some previous value and never updates. same with the voltage and other fields.

**Reply by mr-manuel** [reply]
If you have problems with the latest `dev` version from my repository, then open a bug issue and fill out all needed data. Please read carefully the instructions to make everything right.
